ui-modules/app-inspector/app/views/main/inspect/summary/summary.template.html

```diff
@@ -86,11 +86,19 @@ <h2 ng-if="vm.entity" class="entity-name">
 
         <div ng-if="vm.location.items">
             <div ng-if="vm.location.items.length !== 0 || !vm.location.multi">
-                <p>This entity is running on <em><ng-pluralize count="vm.location.items.length" when="{'0': 'no locations', 'one': '{} location', 'other': '{} locations'}"></ng-pluralize></em></p>
+                <p>This entity is associated with <em><ng-pluralize count="vm.location.items.length" when="{'0': 'no locations', 'one': 'this location ', 'other': 'these locations'}"></ng-pluralize></em></p>
                 <ul>
                     <li ng-repeat="item in vm.location.items track by item.id">
-                        <strong>{{item.config['spec.final'] | specToLabel:vm.metadata}}</strong>
-                        <span ng-if="item.config.availabilityZone">in <code>{{item.config.availabilityZone}}</code> availability zone</span>
+                        <span ng-if="item.id"> Location Id: <strong>{{item.id | specToLabel:vm.metadata}}</strong> </span>
+                        <ul>
+                            <li ng-if="item.name"> <span> Name:  {{item.name | specToLabel:vm.metadata}}</span> </li>
+                            <li ng-if="item.config['spec.final']"> <span> Environment:  {{item.config['spec.final'] | specToLabel:vm.metadata}}</span> </li>
```

Comment:
Does `spec.final` contains the environment? if yes, good to me, just checking

Reply:
Yes, it returns `vphere` for VMware and ` jclouds:aws-ec2` for AWS.